kore/app/share/GlobalMain.hs

```diff
@@ -511,12 +512,14 @@ parseDefinition :: FilePath -> Main ParsedDefinition
 parseDefinition = mainParse parseKoreDefinition
 
 mainParse
-    :: (FilePath -> String -> Either String a)
+    :: (FilePath -> Text -> Either String a)
     -> String
     -> Main a
 mainParse parser fileName = do
     contents <-
-        clockSomethingIO "Reading the input file" $ liftIO $ readFile fileName
+        Text.readFile fileName
+        & liftIO
+        & clockSomethingIO "Reading the input file"
```

Comment:
@ttuegel Why are these the other way round now?

Reply:
I changed it from `($)` to `(&)` so that the action (`Text.readFile`) would appear before the wrappers.